NUService.js

```diff
@@ -139,7 +139,12 @@ export default class NUService extends NUObject {
         return `XREST ${btoa(`${this.userName}:${(this.APIKey || this.password)}`)}`;
     }
 
-    buildURL(entity, RESTResourceName, parentEntity) {
+    /*
+      Since R13 introduced a cloneFrom attribute for one NUObject, 
+      an entity being cloned has entity.ID but it shouldn't be built into the url,
+      notToIncludeEntityID flag is to determine if url needs to include entity.ID or not.
+    */
+    buildURL(entity, RESTResourceName, parentEntity, notToIncludeEntityID) {
```

Comment:
Do we have only one caller to buildURL, I see that you have passed true at line 363, are other calls also present ?  If they are not modified, that means that notToIncludeEntityID be false ?